docs/README.osx.md

```diff
@@ -0,0 +1,193 @@
+- command-line with xcodebuild
+- command-line with make
+
+Kodi for Mac is composed of a main binary with numerous dynamic libraries and
+codecs that support a multitude of music and video formats.
+
+The minimum version of OSX you need to run(!) Kodi is 10.7 atm.
+
+- On Mavericks (OSX 10.9.x) we recommend using Xcode 6.1.
+- On Yosemite (OSX 10.10.x) we recommend using Xcode 6.4.
+- On El Capitan (OSX 10.11.x) we recommend using Xcode 7.x or Xcode 8.x.
+- On Sierra (macOS 10.12.x) we recomment using Xcode 8.x.
+
+**ATTENTION**: You need to download and install XQuartz from <https://xquartz.macosforge.org/landing/>
+since its not part of macOS anymore since 10.8.
+
+## 2 Getting the source code
+
+    cd $HOME
+    git clone git://github.com/xbmc/xbmc.git Kodi
+
+## 3 Install build dependencies
+
+### 3.1 Install Xcode
+
+Install the Xcode version recommended for your macOS version. You can download
+it either from the macOS AppStore (Xcode) or from the Apple Developer Homepage.
+
+As far as we know the compilation for macOS should work with the following
+constellations of Xcode and macOS versions (to be updated once we know more):
+
+1 XCode 6.0.1 against OSX SDK 10.9 (M)
+2 XCode 6.1.0 against OSX SDK 10.10 (Y)
+3 XCode 6.2.0 against OSX SDK 10.10 (Y)
+4 XCode 6.3.0 against OSX SDK 10.10 (Y)
+5 Xcode 6.4.0 against OSX SDK 10.10 (Y)
+6 Xcode 7.x against OSX SDK 10.11 (EC)
+7 Xcode 8.0 against OSX SDK 10.12 (EC)
+8 Xcode 8.0 against OSX SDK 10.12 (S)
+
+### 3.2 Install Kodi build depends
+
+Kodi requires a set of build dependencies to be built and installed before you
+will be able to build the Kodi main binary. These often just called *depends*
+are installed using the commands described below (with the latest iOS SDK found
+on your system).
+
+In order to speedup compilation it is recommended to use `make -j$(getconf
+_NPROCESSORS_ONLN)` instead of `make` to compile on all available processor
+cores. The build machine can also be configured to do this automatically by
+adding `export MAKEFLAGS="-j(getconf _NPROCESSORS_ONLN)"` to your shell config
+(e.g. `~/.bashrc`).
+
+#### 3.2.a Compiling as 64 bit libraries (recommended for most users)
+
+    cd $HOME/Kodi
+    cd tools/depends
+    ./bootstrap
+    ./configure --host=x86_64-apple-darwin
+    make
+
+#### 3.2.b Compiling as 32 bit libraries
+
+    cd $HOME/Kodi
+    cd tools/depends
+    ./bootstrap
+    ./configure --host=i386-apple-darwin
+    make
+
+#### 3.3.c Advanced topics
+
+The dependencies are built into `tools/depends` and installed into
+`/Users/Shared/xbmc-depends`.
+
+**ADVANCED developers only**: If you want to specify an macOS SDK version (if
+multiple versions are installed) - then append it to the configure line
+above. The example below would use the macOS SDK 10.9:
+
+    ./configure --host=arm-apple-darwin --with-sdk=10.9
+
+Ensure that you also adapt the Xcode project to use this SDK version.
```

Comment:
This needs to be adapted in whatever way this needs to be done now with cmake generation...